Dockerfile.dev

```diff
@@ -2,15 +2,17 @@ FROM golang:1.8-alpine
 
 WORKDIR /go/src/github.com/NYTimes/video-captions-api
 
-RUN apk -qq update && apk -qq add --no-cache git mercurial bzr && go get github.com/pilu/fresh
+RUN apk -qq update && apk -qq add --no-cache git mercurial bzr && go get github.com/pilu/fresh && go get github.com/Masterminds/glide
```

Comment:
Unrelated to your change: you can remove ``apk update`` if you're using ``apk add --no-cache``.